erpnext/healthcare/doctype/inpatient_record/inpatient_record.json

```diff
@@ -404,14 +404,15 @@
    "permlevel": 1
   },
   {
-   "fieldname": "discharge_date",
-   "fieldtype": "Date",
+   "fieldname": "discharge_datetime",
```

Comment:
If you are changing the field name, you will have to write a patch for renaming.

Reply:
I have added the patch for renaming. Please check.